go.mod

```diff
@@ -2,26 +2,26 @@ module github.com/kiegroup/kogito-cloud-operator
 
 require (
 	github.com/RHsyseng/operator-utils v0.0.0-20200304191317-2425bf382482
-	github.com/coreos/prometheus-operator v0.38.1-0.20200424145508-7e176fda06cc
+	github.com/coreos/prometheus-operator v0.40.0
```

Comment:
@sutaakar could you please double check why we need this dependency? Normally people are using their `log` package meaning to use the local `log` var

Reply:
Actually it seems that we may not need the prometheus dependency declared. The `ServiceMonitor` seems to be created as a part of reconciliation.

Reply:
So, the dependency is needed for `ServiceMonitor` declaration.
Just it seems that we can remove `PrometheusCli` from https://github.com/kiegroup/kogito-cloud-operator/blob/master/pkg/client/client.go#L62

Reply:
created a separate PR for PrometheusCli removal - https://github.com/kiegroup/kogito-cloud-operator/pull/579